scripts/update_lints.py

```diff
@@ -70,8 +92,40 @@ def get_clippy_lints():
     return merged_lints
 
 
-if __name__ == "__main__":
-    output = [{"name": l[0], "group": l[1], "rustc": True} for l in get_rustc_lints()] + \
-             [{"name": l[0], "group": l[1], "rustc": False} for l in get_clippy_lints()]
+class LintsUpdater(UpdaterBase):
+
+    def _update_locally(self):
+        lints_rustc = get_rustc_lints()
+        lints_rustc.sort(key=lambda item: (not item[1], item[0]))
+
+        text = ""
+        for i in lints_rustc:
+            text += "    Lint(\"{}\", {}),\n".format(i[0], str(i[1]).lower())
+
+        with open(RUSTC_LINTS_PATH, "w") as f:
+            f.write(HARDCODED_PART_RUSTC + text + ")")
+
+        lints_clippy = get_clippy_lints()
+        lints_clippy.sort(key=lambda item: (not item[1], item[0]))
+
+        text = ""
+        for i in lints_clippy:
+            text += "    Lint(\"{}\", {}),\n".format(i[0], str(i[1]).lower())
```

Comment:
nit: it seems it can be replaced with `text = ",\n".join(f"    Lint(\"{item[0]}\", {str(item[1]).lower()})" for item in lints_clippy)`